controller/service.go

```diff
@@ -12,24 +12,30 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+//Service is a type of every method
 type Service struct {
 	Database           *mongo.Database
 	BirthdayCollection *mongo.Collection
+	pb.UnimplementedBirthdayFunctionsServer
 }
 
-type BirthdayStruct struct {
+//BirthdayModel is a model for coverting an object for type: birthdayObject
+type BirthdayModel struct {
 	Name           string
 	Date           string
 	PersonalNumber string
 	ID             string `bson:"_id" json:"id,omitempty"`
 }
 
+//NewService is used each time i create a method
 func NewService(url string) *Service {
 	s := &Service{}
+	s.BirthdayCollection = mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
```

Comment:
"birthdays" and "birthdayDB" should be a const

Reply:
done